### PR TITLE
[-] FO : Avoid displaying payment message

### PR DESCRIPTION
**Pull request by gRoussac**

### CODE REVIEW

File: themes/default-bootstrap/order-detail.tpl [hunk 1]
```diff
@@ -406,6 +406,7 @@
 			</thead>
 			<tbody>
 			{foreach from=$messages item=message name="messageList"}
+				{if !(!$message.id_customer && !$message.id_employee)}
```
Comment (PhpMadman) [inline themes/default-bootstrap/order-detail.tpl]: should it really be **if !(** ?
Can you just do if $message.id_customer && $message.id_employee

Or am I missing something?

**Comment by gRoussac** [comment]
Hi,

I tried to write it simply, message.id_customer can be 0 and $message.id_employee can be 0 but not at tha same time. there must be a smarter way to write it....was the end of the day...

Regards

**Comment by PhpMadman** [comment]
Hmm. Perhaps
if (customer || employee) && !(customer && employee)

Checking that atlest one of them is not 0, and then a NOT check if both are not 0.

**Comment by gRoussac** [comment]
Hi,

customer && employee is ok so ....

**Comment by gRoussac** [comment]
Weel I rather isolate the exception, yours is hard to read too...=)

**Comment by PhpMadman** [comment]
I would say two if statements then. One to check that at least one of them is not 0, and a second to check that both is not 0.

That is most readable.

Or maybe if (!customer && employee) || (!employee && customer) ?

**Pull request closed without merging**